recipe/meta.yaml

```diff
@@ -46,6 +46,7 @@ test:
     {% endfor %}
     - test -f ${PREFIX}/include/bzlib.h  # [unix]
     - test -f ${PREFIX}/lib/libbz2.a  # [unix]
+    - test -f ${PREFIX}/lib/libbz2.so.1.0  # [unix]
```

Comment:
I'd rather not use Makefile-libbz2 on osx,
but instead link the dylib yourself:

```
if test `uname` = "Linux"
then
  make -f Makefile-libbz2_so CFLAGS="$CFLAGS"
  cp -d libbz2.so.* $PREFIX/lib/
else
  gcc -shared -Wl,-install_name -Wl,libbz2.dylib -o libbz2.so.1.0.6.dylib blocksort.o huffman.o crctable.o randtable.o compress.o decompress.o bzlib.o
  ln -s libbz2.${PKG_VERSION}.dylib libbz2.dylib
fi
```